graspologic/embed/mase.py

```diff
@@ -87,6 +87,10 @@ class MultipleASE(BaseEmbedMulti):
     scores_ : array, shape (n_samples, n_components, n_components)
         Estimated :math:`\hat{R}` matrices for each input graph.
 
+    singular_values_ : array, shape (n_components) OR length 2 tuple of arrays
+        If input graph is undirected, equal to the singular values of the concatenated adjacency spectral embeddings.
+        If input graph is directed, :attr:`singular_values_` is a tuple of length 2, where :attr:`singular_values_[0]` corresponds to the concatenated left
```

Comment:
make sure this is <88 chars? if it is please ignore/resolve this comment.

Reply:
Will fix